config/routes.rb

```diff
@@ -307,6 +282,34 @@
     resources :debates, only: [:show]
   end
 
+  scope "(:participatory_process_id)" do
```

Comment:
Maybe instead of having this optional, we could create a specific `match` rule that redirects right at the router layer: http://api.rubyonrails.org/classes/ActionDispatch/Routing/Redirection.html